src/app/common/CodeBlock.vue

```diff
@@ -78,7 +73,6 @@ const reformattedCode = computed(() => props.language === 'yaml' ? reformatYaml(
 async function handleCodeBlockRenderEvent({ preElement, codeElement, language, code }: CodeBlockEventData): Promise<void> {
   isProcessing.value = true
 
-  await nextTick()
```

Comment:
It is no longer required because I addressed the need for it in KCodeBlock where it's adequate. Consuming components shouldn’t care for this sort of timing detail: the correct way to do this was to delay the event until the rendering actually happened.

Reply:
Ah thanks, maybe thats where I saw it last, oh wow ok I just noticed the `isProcessing` thing also, interesting